0164_remove_inventorysource_update_on_project_update.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.13 on 2022-05-17 20:47
+
+from django.db import migrations
+
+def forwards(apps, schema_editor):
+    InventorySource = apps.get_model('main', 'InventorySource')
+    sources = InventorySource.objects.filter(update_on_project_update=True)
+    for src in sources:
+        if src.update_on_launch == False:
+            src.update_on_launch = True
+            src.save(update_fields=['update_on_launch'])
+        proj = src.source_project
+        if proj and proj.update_on_launch == False:
+            proj.update_on_launch = True
+            proj.save(update_fields=['update_on_launch'])
```

Comment:
This is the only change that might be somewhat consequential to the user. I would probably put a warning level log here. This is also a big item for the release notes.